app/models.py

```diff
@@ -1004,6 +1009,7 @@ def serialize(self):
                 for key in self._as_utils_template().placeholders
             },
             "postage": self.postage,
+            "broadcast_data": self.broadcast_data
```

Comment:
yeah this was one thing i wanted to chat about - template.serialize is used in a few places.

three options i can think of:

1. return `broadcast_data` in api calls. Won't need code change in the client libs, but may need docs changes.
1. create a new `template.serialize_for_v2_endpoint` or similar that doesn't include this field
1. only include the _key_ if the template_type is broadcast (so rather than being null, it just doesn't exist)

I think I prefer the second option out of these. Thoughts?

Reply:
Option 2 is good because the public API endpoints should be different to our app to app endpoints.

Reply:
turns out we don't use `template.serialize` anywhere else anyway (we use marshmallow schemas), so i just renamed it to `serialize_for_v2` for clarity